spec/rmagick/image/read_spec.rb

```diff
@@ -26,9 +26,9 @@
     end
   end
 
-  describe 'issue #483' do
+  describe 'issue #483', supported_after('6.9.0') do
+    # On Windows platform, it causes SEGV with ImageMagick 6.8.x
```

Comment:
Maybe make it more clear that this is happening due to the use of a recent Ghostscript version?

Reply:
Now, chocolatey will install ghostscript v9.50 (https://chocolatey.org/packages/Ghostscript)

Reply:
What I meant is that we should writing something like "SEGV because old ImageMagick is not compatible with new Ghostscript". And this is not a Windows only issue.

Reply:
OK, I see. need to fix the comment in spec